aiven/client/cli.py

```diff
@@ -1995,6 +2000,47 @@ def _get_service_project_vpc_id(self):
             project_vpc_id = self.args.project_vpc_id
         return project_vpc_id
 
+    @arg.project
+    @arg.json
+    @arg("--project-vpc-id", required=True, help=_project_vpc_id_help)
+    @arg("--peer-cloud-account", required=True, help=_peer_cloud_account_help)
+    @arg("--peer-resource-group", help=_peer_resource_group_help)
+    @arg("--peer-vpc", required=True, help=_peer_vpc_help)
+    @arg("--peer-region", help=_peer_region_help)
+    @arg("cidrs", nargs="+", metavar="CIDR")
+    def vpc__user_peer_network_cidr__add(self):
+        """Add one ore more peer network CIDRs to a VPC"""
+        project_name = self.get_project()
+        add_base = {
+            "peer_cloud_account": self.args.peer_cloud_account,
+            "peer_vpc": self.args.peer_vpc,
+        }
+        if self.args.peer_resource_group is not None:
+            add_base["peer_resource_group"] = self.args.peer_resource_group
+        if self.args.peer_region is not None:
+            add_base["peer_region"] = self.args.peer_region
```

Comment:
This isn't listed as supported property in https://api.aiven.io/doc/#api-Project-VpcPeeringConnectionUpdate

Reply:
Good catch. Removed. Should be added to the Aiven API, though currently it doesn't matter much as no `user_peer_network_cidr` compatible peering connection types support it.